doc/cookbook/source/examples/classifier/kmeans.rst

```diff
@@ -0,0 +1,47 @@
+==================
+:math:`k`-means clustering
```

Comment:
Minors: Capital K please. Also, can you avoid the math in the header (only there)